### ISSUE TITLE
Activity Panel: Notices Button not functioning

### ISSUE THREAD

**Issue by derekakelly** [woocommerce/woocommerce-admin#2520]
**Describe the bug**
The Notices button in the Activity Panel isn't functional for me.

**To Reproduce**
When i click on the notices button, all it does is add a small purple line underneath it. 

**Expected behavior**
I expected something to come out in a similar way to the other buttons which all function as expected.

**Screenshots**
![notices-button](https://user-images.githubusercontent.com/5824837/60210357-affaac00-982a-11e9-80a1-c752385f50af.png)

**Desktop (please complete the following information):**

Windows 10 tested with both edge and chrome

**Comment by rrennick** [comment]
@derekakelly Thanks for reporting this.

- Was this with the new 0.14.0 version? 
- Do you see any script errors in your browser console?

**Comment by derekakelly** [comment]
Hi Ron,

Yes, I just updated to 0.14.0

I looked in my Console and don't see any errors that were related.

This is what i did get, but only when i resized console. 
![java console](https://user-images.githubusercontent.com/5824837/60212218-af641480-982e-11e9-97db-b3dadb6692cf.png)

I also checked my debug.log and didn't find anything

Is this related to Jetpack? Because I don't use Jetpack.

**Comment by rrennick** [comment]
@derekakelly Can you try two quick tests

- Navigate to a different WC screen, click on the Notice button once the page loads
- Navigate to a second screen, scroll down then click on the Notice button

The behaviour I'm seeing is that the Notice button works when the page is scrolled to top. When the page is scrolled down the button does show the notices but does not scroll up to make them visible.

**Comment by derekakelly** [comment]
@rrennick I've followed your instructions but I'm still not getting anything.

**Comment by timmyc** [comment]
@derekakelly I'm not able to reproduce the issue on Firefox myself - by chance could you inspect the dom for the div with the id of `wp__notice-list` and see if there is indeed some notices being output there?

Alternatively - if you visit a non-woo page, like perhaps the post listing page `/wp-admin/edit.php` are you seeing any admin notices there?

FWIW I created a test notice using the following:

```
<?php
/**
 * Plugin Name: Just A Notice
 * Plugin URI: https://woot.com
 * Description: Just a notice
 * Author: WooCommerce
 * Author URI: https://woocommerce.com/
 * Domain Path: /languages
 * Version: 0.13.2
 */

function sample_admin_notice__error() {
	$class = 'notice notice-error';
	$message = 'just a notice';

	printf( '<div class="%1$s"><p>%2$s</p></div>', esc_attr( $class ), esc_html( $message ) ); 
}
add_action( 'admin_notices', 'sample_admin_notice__error' );
```

**Comment by derekakelly** [comment]
I've setup a staging site and disabled all plugins, then by re-adding them one by one, I found the issue came back while the WooCommerce Order Status Manager plugin by skyverge was activated.

Until that plugin was loaded, there are only 3 buttons i in the Activity Panel that are visible, the Notices button is hidden.

@timmyc I found the section you were referring to but i couldn't find any useful information.
![dom](https://user-images.githubusercontent.com/5824837/60321303-6bfec880-994a-11e9-9a6c-068ba95881c0.png)

**Comment by tommyshellberg** [comment]
@derekakelly very strange, I was able to reproduce this with Order Status Manager enabled but not after deactivating it and reactivating it. Everything with WC-Admin functionality and markup looked fine. It was toggling the `show/hide` class but there simply wasn't any notices in the HTML markup. Just like in your screenshot, there should be(if there are notices in the shop) some <div class="error"> elements.

![https://cld.wthms.co/8SP3We](https://cld.wthms.co/8SP3We+)
Image link: https://cld.wthms.co/8SP3We

**Comment by timmyc** [comment]
@derekakelly I'm still not able to reproduce the bug with Order Status Manager installed too. 

Based off your screenshot though, it looks like the empty divs for `js-wc-cart-notices-admin-notice-placeholder` is the cause of the issue. Do you have WooCommerce Cart Notices installed/enabled? When I activated that extension, I can reproduce the bug. I'll work on a fix.

**Comment by derekakelly** [comment]
@timmyc Yes, I have WooCommerce Cart Notices installed, but not activated on my staging site.

**Comment by timmyc** [comment]
@derekakelly interesting, wondering why that DOM element is still there then. Still I think the solution I have in #2552 should fix up the bug you are seeing. Thanks again for the report and for all the help in troubleshooting!

**Comment by timmyc** [comment]
@derekakelly I just merged in the fix, so this issue was closed out during that process. The fix will go out in v0.15.0 in about a week from now.
